ChannelDataExtractor.py

```diff
@@ -0,0 +1,33 @@
+import settings
+from DataExtractor import DataExtractor
+
+
+class ChannelDataExtractor(DataExtractor):
+    __key = settings.API_key
+    __part = 'statistics'
```

Comment:
hardcode. There is a real chance for parts to be changed in some cases. It'll be better to make them constructor parameters with default value.